lib/backburner/utils.ts

```diff
@@ -31,7 +31,7 @@ export function getOnError(options) {
 export function findItem(target, method, collection) {
   let index = -1;
 
-  for (let i = 0, l = collection.length; i < l; i += 3) {
+  for (let i = 0, l = collection.length; i < l; i += 4) {
```

Comment:
Seems like this change was what was moved to the other PR, right?

Reply:
Yes, shall i remove it?

Reply:
Yes, please 😄 

I don't fully grok this ATM because I thought we already merged the other PR.  Lets remove because I don't think this change is related to the other changes.